perf-tool/src/agentOptions.cpp

```diff
@@ -175,34 +180,23 @@ void modifyObjectAllocEvents(const std::string& function,const std::string& comm
     }
 }
 
-void modifyMonitorStackTrace(const std::string& function, const std::string& command)
-{
-    /* enable stack trace */
-    if (!command.compare("start"))
-    {
-        setMonitorStackTrace(true);
-    }
-    else if (!command.compare("stop"))
-    {
-        setMonitorStackTrace(false);
-    }
-    else
-    {
-        invalidCommand(function, command);
-    }
-}
```

Comment:
This seems to be unrelated to the method enter/exit events; is this deleting the ability for the user to send `monitorStackTrace`?

Reply:
Yes. The stacktrace option is sent with the "start" command.
The exiting code actually did nothing